code/parachain/frame/composable-maths/src/dex/constant_product.rs

```diff
@@ -529,3 +529,29 @@ pub fn compute_deposit_lp_<T: PerThing>(
 
 	Ok(ConstantProductAmmValueFeePair { value: issued, fee })
 }
+
+/// Computes the required deposit to recieve a minimum amount of LPT
+///
+/// NOTE: When considering accounting for fees - the result of this function may produce a result
+/// slightly lower than the minimum LP to mint.
+///
+/// # Parameters
+/// * `p_supply` - Existing supply of LP tokens
+/// * `p_issued_min` - Minumum LPT a user wants from thier investment
+/// * `b_k` - Balance of token `k`
+/// * `f` - Fee
+pub fn compute_deposit_for_min_lp<T: PerThing>(
+	p_supply: u128,
+	p_issued_min: u128,
+	b_k: u128,
+	f: T,
+) -> ConstantProductAmmResult<ConstantProductAmmValueFeePair> {
```

Comment:
This function is unused but will have value in the future when LP tokens are more directly used by users. This will allow users to request an amount of LPT without needing to manually calculate the liquidity to provide.